projects/fsastorefrontlib/src/cms-components/my-account/quote/quote-comparison/quote-comparison.component.ts

```diff
@@ -0,0 +1,133 @@
+import { Component, OnDestroy, OnInit } from '@angular/core';
+import { QuoteService } from '../../../../core/my-account/facade/quote.service';
+import { FSTranslationService } from '../../../../core/i18n/facade/translation.service';
+import { Observable, Subscription } from 'rxjs';
+import { tap } from 'rxjs/operators';
+import {
+  CategoryComparisonConfig,
+  QuoteComparisonConfig,
+} from 'projects/fsastorefrontlib/src/core/quote-comparison-config/quote-comparison-config';
+import { ChangeDetectionStrategy } from '@angular/core';
+import {
+  FSCart,
+  InsuranceQuote,
+  OneTimeChargeEntry,
+} from '../../../../occ/occ-models/occ.models';
+import {
+  LanguageService,
+  RoutingService,
+  TranslatePipe,
+} from '@spartacus/core';
+
+@Component({
+  selector: 'cx-fs-quote-comparison',
+  templateUrl: './quote-comparison.component.html',
+  changeDetection: ChangeDetectionStrategy.OnPush,
+})
+export class QuoteComparisonComponent implements OnInit, OnDestroy {
+  private readonly paynowBillingTimeCode = 'paynow';
```

Comment:
Perhaps move this to some global constant since it's used in several places?

Reply:
Where should I place that global constant?

Reply:
Maybe create separate file general.config or something like that, which would then be referenced in all files using this 'paynow' string? Or maybe just place it in one of those config files you've created in this task, and then reference it from there?